k-distribution/include/ktest.mak

```diff
@@ -38,7 +38,9 @@ KAST_TESTS?=$(wildcard $(TESTDIR)/*.kast)
 # default KOMPILE_BACKEND
 KOMPILE_BACKEND?=llvm
 
-CHECK=| diff -
+CHECK?=| diff -
+REMOVE_PATHS=sed 's!'`pwd`'/\(\./\)\{0,1\}!!g'
+CHECK_ERRORS=2>&1 | $(REMOVE_PATHS) | diff -
```

Comment:
unfortunately this will break the update-results target, because sed will not be run in that case and you will populate the output files with results that will not pass tests. The sed needs to be outside the CHECK variable.